core/controllers/resources.py

```diff
@@ -86,3 +87,24 @@ def get(self, exploration_id, asset_type, encoded_filename):
             self.response.write(raw)
         except:
             raise self.PageNotFoundException
+
+
+class PromoBarHandler(base.BaseHandler):
+    """The handler for checking if Promobar is enabled and fetching
+    promobar message.
+    """
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.open_access
+    def get(self):
+        if constants.ENABLE_PROMO_BAR:
+            promo_bar_enabled = config_domain.PROMO_BAR_ENABLED.value
+            promo_bar_message = config_domain.PROMO_BAR_MESSAGE.value
+        else:
+            promo_bar_enabled = False
+            promo_bar_message = ''
```

Comment:
Since we will check for `ENABLE_PROMO_BAR` on frontend we don't need to do that here, so you can probably write `promo_bar_enabled = config_domain.PROMO_BAR_ENABLED.value or False`

Reply:
Done.